scripts/calibrateArmDac.sh

```diff
@@ -34,71 +34,80 @@ then
     return
 fi
 
-echo -e "ChamberName\tscandate\tCFG_THR_ARM_DAC" 2>&1 | tee listOfScanDates_calibrateArmDac_${DETECTOR}.txt
+echo -e "ChamberName\tscandate\tCFG_THR_ARM_DAC" 2>&1 | tee ${DATA_PATH}/${DETECTOR}/armDacCal/${scandate}/listOfScanDates_calibrateArmDac_${DETECTOR}.txt
+
+scandate=$(date +%Y.%m.%d.%H.%M)
+mkdir -p ${DATA_PATH}/${DETECTOR}/armDacCal/${scandate}
+ln -s ${DATA_PATH}/${DETECTOR}/armDacCal/${scandate} ${DATA_PATH}/${DETECTOR}/armDacCal/current
```

Comment:
You'll need to add the option to force create the link if already exists.  Otherwise it won't be re-created and it will always overwrite the data that `current` points too. e.g. switch to:

`ln -sf`

With option `f`